cloud/app/web_components/robot-agent-device.jsx

```diff
@@ -476,38 +493,66 @@ const Device = (props) => {
   //     // using console.log to get colors from escape sequences in output:
   //     console.log(response));
 
+  const errorCount = latestVersionData?.status?.logs?.errorCount?.['@transitive-robotics']?.['robot-agent'] || 0;
   log.debug({latestVersionData, packages, toast});
 
   return <div>
-    <div style={styles.row}>
+    <div style={styles.row} className='position-relative'>
       <OSInfo info={latestVersionData?.info}/>
-      {latestVersionData.status?.heartbeat &&
-          <Heartbeat heartbeat={latestVersionData.status.heartbeat}/>
-      } <span style={styles.agentVersion} title='Transitive agent version'>
-        v{latestVersion}
-      </span>&nbsp;&nbsp; <ActionLink disabled={inactive}
-        onClick={() => runCommand('ping', {timestamp: Date.now()},
-          (time) => setToast({ title: 'Pong!',
-            body: `Server time: ${new Date(time)}`})
-        )}
-        onContextMenu={() => ping(10)} // hidden feature: right-click
-      >
-        Ping
-      </ActionLink>&nbsp;&nbsp; <ActionLink onClick={restartAgent} disabled={inactive}>
-        Restart agent
-      </ActionLink>&nbsp;&nbsp; <ActionLink onClick={
-          () => runCommand('stopAll', {}, log.debug)} disabled={inactive}>
-        Stop all capabilities
-      </ActionLink>&nbsp;&nbsp; <ConfirmedButton onClick={clear}
-        explanation={explanation} question='Remove device?'>
-        Remove device
-      </ConfirmedButton>&nbsp;&nbsp; <LogButtonWithCounter
-          text="Get log"
-          mqttSync={mqttSync}
-          versionPrefix={versionPrefix}
-          packageName='@transitive-robotics/robot-agent'
-          errorCount={latestVersionData?.status?.logs?.errorCount?.['@transitive-robotics']?.['robot-agent']}
-        />
+      <div className='d-flex align-items-baseline my-2 my-lg-0'>
+        <span className='d-flex align-items-center'>
+          {latestVersionData.status?.heartbeat &&
+              <Heartbeat heartbeat={latestVersionData.status.heartbeat}/>
+          } <span style={styles.agentVersion} title='Transitive agent version'>
+            v{latestVersion}
+          </span>
+        </span>
+        <Dropdown className={`position-absolute top-0 end-0 me-2`}>
+          <Dropdown.Toggle variant='link' size='sm' bsPrefix='_' className='position-relative'>
+            <FaEllipsisH style={{ color: '#000' }}/>
+            {errorCount > 0 && <Badge pill bg='danger' className='position-absolute ms-1'
+              style={{
+                fontSize: '0.5em',
+              }}
+              title={`There are ${errorCount} errors.`}>
+              {errorCount}
+            </Badge>}
+          </Dropdown.Toggle>
+          <Dropdown.Menu>
+            <ActionLink disabled={inactive}
```

Comment:
Why are we still using ActionLinks instead of `Dropdown.Item`'s? Does the hover effect still work with these? I cannot see the hover effect on my screen (it's not the best monitor), but I can't see it for the capability menu either but I know it's there.